src/controller/userControllers.js

```diff
@@ -86,18 +87,102 @@ class userController {
                 return false
               }
               const user = await Model.User.create(theUser); 
-              const Options ={
-                email:req.body.email,
-                password:generatedPassword,
+              const options ={
+                  userEmail:`${email}`,
+                  subject: 'Phantom Registration Successful',
```

Comment:
Just a challenge, it's not required: detect the language for the user and provide a french email in case they use french.